app/views/styleguide/hero.html.erb

```diff
@@ -0,0 +1,106 @@
+<h1>UI Components</h1>
+<h2>Hero section</h2>
+<h3 class="styleguide__subheading">Default title</h2>
+<div class="styleguide__example">
+  <div class="hero styleguide__hero">
+    <h1 class="hero__heading">
+      Main heading of the page
+    </h1>
+  </div>
+</div>
+<h3 class="styleguide__subheading">Code</h3>
+<xmp>
+  <div class="hero">
+    <h1 class="hero__heading">
+      Main heading of the page
+    </h1>
+  </div>
+</xmp>
+<h3 class="styleguide__subheading">Title contains highlighted content</h2>
+<div class="styleguide__example">
+  <div class="hero styleguide__hero">
+    <h1 class="hero__heading">
+      This heading <span class="hero__heading-strong">contains some bold copy</span>
+    </h1>
+  </div>
+</div>
+<h3 class="styleguide__subheading">Code</h3>
+<xmp>
+  <div class="hero">
+    <h1 class="hero__heading">
+      This heading 
+      <span class="hero__heading-strong">contains some bold copy</span>
+    </h1>
+  </div>
+</xmp>
+
+<h2>Colour variants</h2>
+<p>Each of the primary colours can be used for the main title background by adding a modifier classname to the hero element.</p>
+
+<h3 class="styleguide__subheading">Primary Blue</h2>
+<div class="styleguide__example">
+  <div class="hero hero--blue styleguide__hero">
+    <h1 class="hero__heading">
+      Main heading of the page
+    </h1>
+  </div>
+</div>
+<h3 class="styleguide__subheading">Code</h3>
+<xmp>
+  <div class="hero hero--blue">
+    <h1 class="hero__heading">
+      Main heading of the page
+    </h1>
+  </div>
+</xmp>
+
+<h3 class="styleguide__subheading">Primary Orange</h2>
+<div class="styleguide__example">
+  <div class="hero hero--orange styleguide__hero">
+    <h1 class="hero__heading">
+      Main heading of the page
+    </h1>
+  </div>
+</div>
+<h3 class="styleguide__subheading">Code</h3>
+<xmp>
+  <div class="hero hero--orange">
+    <h1 class="hero__heading">
+      Main heading of the page
+    </h1>
+  </div>
+</xmp>
+
+<h3 class="styleguide__subheading">Primary Purple</h2>
+<div class="styleguide__example">
+  <div class="hero hero--purple styleguide__hero">
+    <h1 class="hero__heading">
+      Main heading of the page
+    </h1>
+  </div>
+</div>
+<h3 class="styleguide__subheading">Code</h3>
+<xmp>
+  <div class="hero hero--purple">
+    <h1 class="hero__heading">
+      Main heading of the page
+    </h1>
+  </div>
+</xmp>
+
+<h3 class="styleguide__subheading">Primary Yellow</h2>
```

Comment:
This has an opening `h3` and a closing `h2` (it's also in a couple places further up)